### PR TITLE
Fixes for simple typos

### PR DESCRIPTION
**Pull request by dprats**
Hi there!

In reading the documentation, I found a few low-impact typos that I figured I could fix while working through the lessons. 

I have learned a lot from this documentation effort so I figured I could contribute back.

### CODE REVIEW

File: docs/core/action-properties.rst [hunk 1]
```diff
@@ -98,7 +98,7 @@ But that's not (yet) a valid TLA+ property. Consider the slightly different prop
 
 As yet more syntactic sugar, we can write ``[](x' = x + 1 \/ UNCHANGED x)`` as ``[][x' = x + 1]_x``. This is called a :dfn:`box action formula`. Box action formulas have a special role in TLA+, as we'll see in :doc:`the next chapter <tla>`. TLC can only check action properties that are box action formulas.
 
-.. tip:: The underscory bit means that we could have written the property as ``[][counter' > counter]_counter``. Expanding all the steps: 
+.. tip:: The underscory bit (`_`) means that we could have written the property as ``[][counter' > counter]_counter``. Expanding all the steps:
```
Comment (hwayne) [inline docs/core/action-properties.rst:101]: In rST you use double backticks for code samples:

```
``_``
```

**Review by hwayne — CHANGES_REQUESTED** [review]
Ah, thank you so much! Only one request to fix an rST formatting issue

File: docs/core/action-properties.rst (hunk 1 above)
Comment (dprats) [reply docs/core/action-properties.rst:101]: ah gotcha. Will fix.

Comment (dprats) [reply docs/core/action-properties.rst]: fixed

**Comment by dprats** [comment]
@hwayne fixed the comment. hope that helps!

**Comment by hwayne** [comment]
Thank you!

**Pull request merged**